grapher/scan.py

```diff
@@ -132,11 +132,22 @@ def pkgToUnit(pkg: Dict) -> List[Unit]:
     if len(test_files) == 0:
         return [unit]
 
+    # Collect files as candidates of modules.
+    modules = []
```

Comment:
Met this issue while testing on `httpie` (https://github.com/jkbrzt/httpie/blob/master/setup.py#L81), it only returns 4 packages, and not include `httpie.input` (it's a file) but in test file (https://github.com/jkbrzt/httpie/blob/master/tests/test_httpie.py#L4) it does import file as a module.